internal/engine/upper_test.go

```diff
@@ -161,21 +130,6 @@ func TestUpper_Up(t *testing.T) {
 	assert.Equal(t, gYaml, state.GlobalYAML)
 }
 
-func TestUpper_UpWatchError(t *testing.T) {
```

Comment:
why did this go away?

Reply:
ahh thank you, I need to add error support to the new fakeMetaWatcher.